audtorch/transforms/transforms.py

```diff
@@ -812,6 +855,45 @@ def __repr__(self):
         return '{0}({1})'.format(self.__class__.__name__, options)
 
 
+class Log(object):
+    r"""Logarithmic transform of an input signal.
+
+    * :attr:`magnitude_boost` controls the non-negative value added to the
+      magnitude of the signal before applying the logarithmus
+
+    Args:
+        magnitude_boost (float, optional): positive value added to the
+            magnitude of the signal before applying the logarithmus.
+            Default: `1e-7`
```

Comment:
Wrong indentation.

Reply:
Is this fixed? Tests are passing for me...

Reply:
Sorry, I think this was my fault.